advanced-achievements-api/src/main/java/com/hm/achievement/utils/PlayerAdvancedAchievementEvent.java

```diff
@@ -172,8 +172,8 @@ public PlayerAdvancedAchievementEventBuilder commandRewards(String[] commandRewa
 			return this;
 		}
 
-		public PlayerAdvancedAchievementEventBuilder itemReward(ItemStack itemReward) {
```

Comment:
This method is part of the public API. It should be deprecated rather than removed. An implementation similar to what is done for `commandMessage` should do the trick.